libs/template/application/service-worker/app/app-builder.ts

```diff
@@ -0,0 +1,6 @@
+import { AppBuilderWithModules } from './app.model';
+import { ModularAppBuilder } from './modular-app-builder';
+
+export function appBuilder(): AppBuilderWithModules {
```

Comment:
Why do we need a copy-paste from `src/app/app-builder.ts`?

Reply:
This is required to drop lit-releated stuff and not including it to the sw-bundle. I was thinking about creation of something like `BasicAppBuilder`, but we are already have the `SimpleAppBuilder` super class, that can confuse. In result I've just extracted minified version to the '@spryker-oryx/application/service-worker' package to keep it in one place with workbox bootstrap

Reply:
Ok, let's keep it as is for now.